sample/java/src/main/java/com/datadog/android/sample/traces/TracesViewModel.java

```diff
@@ -102,6 +100,12 @@ protected Result doInBackground(Void... voids) {
 
             try {
                 Response response = okHttpClient.newCall(request).execute();
+                ResponseBody body = response.body();
+                if (body != null) {
+                    String content = body.string();
+                    Log.d("Response", content);
```

Comment:
Why logging this into console ?

Reply:
This is simply to consume the response, otherwise the timings are never complete and the resource event not sent.